pages/linux/lxi.md

```diff
@@ -0,0 +1,24 @@
+# lxi
+
+> Control LXI compatible instruments such as oscilloscopes.
+> More information: <https://github.com/lxi-tools/lxi-tools>.
+
+- Discover LXI devices on available networks:
+
+`lxi discover`
+
+- Capture a screenshot using plugin autodetection:
+
+`lxi screenshot --address {{ip_address}}`
+
+- Capture a screenshot using specifying a plugin:
```

Comment:
How about:

```suggestion
- Capture a screenshot using the specified plugin:

Reply:
I am tempted to accepted it but, could you please consider a shorter version like this?
```suggestion
- Capture a screenshot using a plugin:
```

Reply:
```suggestion
- Capture a screenshot using a specified plugin:
```

I prefer a to the

Reply:
We normally use specific and specified depending on the situation.

Specific may fit better here actually